client/src/views/account/RecoverAccountPage.vue

```diff
@@ -436,6 +470,11 @@ async function backToLogin(): Promise<void> {
   box-shadow: var(--parsec-shadow-soft);
   padding: 2rem;
   gap: 1rem;
+  position: relative;
+
+  .input-edit-button {
+    right: 2.25rem;
+  }
```

Comment:
I guess this css class interacts with the one in `_inputs.scss`, and the button does nothing as a result